manifests/vcsscm/git.pp

```diff
@@ -20,6 +22,26 @@
           ensure => $ensure,
         }
       }
+
+      if $bash_completion {
+        package { 'bash-completion':
+          ensure => $ensure,
+        }
+      }
+
+      if $bash_prompt {
```

Comment:
Does bash prompt require bash_completion to work? Maybe we should require it?
Not sure... I guess some may already have it like on Ubuntu Desktop and may not want to manage it here.

Reply:
The prompt and the tab completion feature are independent.

Reply:
does /etc/bash_completion.d come with core bash? Or does the bash-completion package add that?